students/rhamersky/Lesson_6/mailroom_part4.py

```diff
@@ -0,0 +1,325 @@
+#!/usr/bin/env python3
+# Description: This program will allow a user to input a donor and a donation.
+# The user will also be able to send out thank you notes to the donors.
+# Developer: Ryan Hamersky
+# Date: 05/13/2018
+# Rev: A - 05/16/2018 updated code to reflect comments made by the instructor in mailroom_part.py.
+#      Update variables to snake_case. Removed unnecessary 'continues' and 'returns'.
+#      Changed 'bool' variable to 'bool_flag'. Updated misleading function names.
+#      Reduced the amount of unnecessary print() statements.
+#
+# Rev: B - 05/19/2018 updated code to reflect comments made by the instructor in mailroom_part3.py.
+
+# -----Data Section-----
+import os
+str_name = str()    # --> User's name
+int_option = int()  # --> User's option.
+tup_donor_names = (["William Gates, III",1,10000.0],["Mark Zuckerberg",1,20000.0],
+                   ["Jeff Bezos",1,1200.0],["Paul Allen",1,5000.0],
+                   ["Mike Scott",1,25.0])  # --> Tuple that stores all donor's information
```

Comment:
You should be using a `dict` for your main data structure at this point in the course and that will simplify a lot of your other code that deals with looking up and adding donors and donations